lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1516,3 +1538,68 @@ def _get_disk_id(node_features: search_space.SetSpace[str]) -> str:
             return features.DiskStandardSSDLRS.get_disk_id()
         else:
             raise LisaException("No disk feature present.")
+
+    @staticmethod
+    def _get_data_disk_size_by_iops(
```

Comment:
`_get_data_disk_size` And move related code to another file like common.py. The platform.py focus on platform.